src/routes/articles.js

```diff
@@ -107,6 +102,39 @@ const getArticle = async (req, res, next) => {
 	}
 }
 
+
+const updateArticle = async (req, res, next) => {
+	const { id } = req.params 
+	const { title, article, image, published } = req.body 
+
+	try {
+		const updatedArticle = await postService.updatePost({
+			title,
+			content : article,
+			image,
+			published,
+			id
+		})
+		if (!updatedArticle) {
+			const errorMessage = ArticleDoesNotExistError.message
```

Comment:
kindly double check what was done in the other pr @beblicarl. This is different

Reply:
when i tried it the message i got was that err.message isn't defined so i had to define it that's why i did it this way

Reply:
I have doubled checked now and its the same, trust me @jocrah hehe